README.md

```diff
@@ -202,6 +202,23 @@ behavior of the omnibox:
        last.
     4. Hitting ESC will close the list of suggestions and clear the field.
 
+## Omnibox Event Bindings
+
+The following &-callback functions are available for binding on the `ngc-omnibox` component in
+response to events:
+
+- `onChosen({choice})`: An expression that's called when a suggestion is chosen. It receives, in its
+scope, access to `choice`, which is the item that was chosen.
```

Comment:
ok, i'll replace them all with "locals". that parlance is familiar to angular devs?